app/helpers/application_helper.rb

```diff
@@ -1,5 +1,11 @@
+require "yaml"
```

Comment:
[question] is this leftover from an earlier version? I don't see why it would be needed